overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -884,6 +884,19 @@ def _check_valid_state(self, state):
         for obj_state in all_objects:
             assert obj_state.is_valid()
 
+    def find_free_counters_valid_for_both_players(self, state, mlp):
+        """Finds all empty counter locations that are accessible to both players"""
+        one_player = state.players[0]
+        other_player = state.players[1]
```

Comment:
Nicer to write `one_player, other_player = state.players` in one line (tuples automatically unpack)